proto/storage/vuln_requests.proto

```diff
@@ -112,7 +121,9 @@ message VulnerabilityRequest {
 
     // 21 to 25 reserved for the updated request type oneof.
     oneof updated_req {
-        DeferralRequest              updated_deferral_req       = 21 [(gogoproto.moretags) = 'search:"-"'];
+        DeferralRequest              updated_deferral_req = 21 [deprecated = true, (gogoproto.moretags) = 'search:"-"'];
+        DeferralUpdate               deferral_update      = 22 [(gogoproto.moretags) = 'search:"-"'];
+        FalsePositiveUpdate          fp_update            = 23;
```

Comment:
Is there a way to block 24-25 aside from this comment and hoping authors and reviewers see it?

Reply:
nit (up to you):  for readability should we consider `false_positive_update`?

Reply:
🤞